spec/seeds/users_seeds.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS users;
+
+-- Table Definition
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  name text,
+  email varchar
```

Comment:
I didn't know about this datatype. When reading a bit more about it, it says it's a variable- length character string. How is that controlled, do you know? Just curious.